lib/content-services/src/lib/common/interfaces/search-configuration.interface.ts

```diff
@@ -27,6 +26,5 @@ export interface SearchConfigurationInterface {
      * @param skipCount The offset of the start of the page within the results list
      * @returns Query body defined by the parameters
```

Comment:
This probably should be updated too

Reply:
it's just the concept here, not the type name